pulumi/aws/anthos/__main__.py

```diff
@@ -251,26 +274,32 @@ def add_namespace(obj):
 if not anthos_host:
```

Comment:
Why are we doing this? This also deserves a comment (if it doesn't already exist).

Reply:
There was a comment, but I beefed it up a bit. the tl;dr is that we need to give the ingress object a hostname, and we default to the loadbalancer name. I note how this impacts TLS if the user moves to real certs and not self-signed.